client/obbatch.go

```diff
@@ -322,8 +322,6 @@ func (b *obBatchExecutor) executeInternal(ctx context.Context) (BatchOperationRe
 		wg.Wait()
 		if len(errArr) != 0 {
 			log.Warn("error occur when execute partition operations")
-			//return newObBatchOperationResult(res), errArr[0], needRetry
-			return nil, errArr[0], needRetry
```

Comment:
I guess removing this error may miss some errors in rpcHeader.
I guess we collect all errors (not the error in optionResult) and put them in errArr.

Reply:
fixed